File: uri/media/media.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    
    private static double pesoA = 3.5;
    private static double pesoB = 7.5;
    
    private static double media( double notaA, double notaB )
    {
        double numMedia;
        numMedia = ((notaA * pesoA) + (notaB * pesoB)) / (pesoA + pesoB);
        return numMedia;
    }
    
    public static void main(String[] args) throws IOException {  
        Scanner input = new Scanner(System.in);
        double a, b, nmedia;
        a = input.nextDouble();
        b = input.nextDouble();
        nmedia = media(a, b);
        System.out.printf("MEDIA = %.5f\n", nmedia);
    }
    
}

File: uri/media/media.ml
let media x y = 
  ((x *. 3.5) +. (y *. 7.5)) /. 11.0 ;;

let notaA = read_float() ;;
let notaB = read_float() ;;
let media_num = media notaA notaB ;;
Printf.printf "MEDIA = %.5f\n"  media_num;;